test/scripts/imgtestlib.py

```diff
@@ -362,12 +369,24 @@ def read_osrelease():
 
 
 def get_osbuild_commit(distro_version):
+    """
+    Get the osbuild commit defined in the Schutzfile for the host distro.
+    If not set, returns None.
+    """
     with open(SCHUTZFILE, encoding="utf-8") as schutzfile:
         data = json.load(schutzfile)
 
     return data.get(distro_version, {}).get("dependencies", {}).get("osbuild", {}).get("commit", None)
 
 
+def get_osbuild_nevra():
```

Comment:
Sorry, I'm new - what does `nevra` stand for?

Reply:
No need to apologise.  I still remember having the exact same question and finding it hard to pin down.
It stands for [name-epoch-version-release-architecture](https://docs.fedoraproject.org/en-US/modularity/core-concepts/nsvca/#_naming_and_identifying_modules_in_fedora), which is the Fedora package naming convention.